docs/docsource/operation/XDM_GUI/file_operations/upload_files_to_buckets.rst

```diff
@@ -0,0 +1,51 @@
+Uploading Files to Buckets
+==========================
```

Comment:
Tiny detail. For consistency with the other titles, I'd change "Uploading" to "Upload".

Reply:
Right you are!